modules/deployment/src/test/java/org/mule/runtime/module/deployment/internal/AbstractDeploymentTestCase.java

```diff
@@ -972,9 +974,13 @@ protected Application findApp(final String appName, int totalAppsExpected) {
   }
 
   protected DefaultMuleDomain createDefaultDomain() {
-    return new DefaultMuleDomain(new DomainDescriptor(DEFAULT_DOMAIN_NAME),
+    DomainDescriptor descriptor = new DomainDescriptor(DEFAULT_DOMAIN_NAME);
+    return new DefaultMuleDomain(descriptor,
                                  new DomainClassLoaderFactory(getClass().getClassLoader())
-                                     .create("domain/" + DEFAULT_DOMAIN_NAME, containerClassLoader,
+                                     .create("domain/" + DEFAULT_DOMAIN_NAME,
+                                             new RegionClassLoader("domainRegion", descriptor,
```

Comment:
wow, good catch! Could this break something or actually stop working something that was working before due to this bug?